stdlib/@python2/ctypes/__init__.pyi

```diff
@@ -150,7 +150,7 @@ def create_unicode_buffer(init: _UnionT[int, Text], size: int | None = ...) -> A
 if sys.platform == "win32":
     def DllCanUnloadNow() -> int: ...
     def DllGetClassObject(rclsid: Any, riid: Any, ppv: Any) -> int: ...  # TODO not documented
-    def FormatError(code: int) -> str: ...
+    def FormatError(code: int | None = ...) -> str: ...
```

Comment:
Can it really be None? Looking at https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Modules/_ctypes/callproc.c#L1316 I think it must be an int or omitted, so we should just write `code: int = ...`.

Reply:
ah, looks like my ide lied to me, pushed a fix